bed/methods.go

```diff
@@ -30,15 +35,22 @@ func (b Bed) UpdateCoord(c string, start int, end int) interface{} {
 
 type BedSlice []*Bed
 
+// Len returns total bed entries within the input bed slice
 func (b BedSlice) Len() int { return len(b) }
 
+// Swap will switch the values of two bed structs inside
+// the slice of beds.
 func (b BedSlice) Swap(i, j int) { b[i], b[j] = b[j], b[i] }
 
+// Push and pop() satisfy the interface for the heap method
+// Push pushes to the heap.
 func (b *BedSlice) Push(x interface{}) {
 	answer := x.(*Bed)
 	*b = append(*b, answer)
 }
 
+// Pop and push() satisfy the interface for the heap method
+// Pop pushes to the heap.
```

Comment:
I think this should be something like "Pop returns a value from the heap" rather than pushing to the heap.

Reply:
One option would be to also mention that the element (along with being returned) is also removed from the heap.